lib/measures/window_enhancement/resources/EC3_lookup.py

```diff
@@ -0,0 +1,218 @@
+# EC3 API Lookup Script
+import requests
+import json
+import re
+from typing import Dict, Any, List
+from pathlib import Path
+import configparser
+from datetime import datetime
+import sys
+import os
+
+script_dir = os.path.dirname(os.path.abspath(__file__))
+repo_root = os.path.abspath(os.path.join(script_dir, "../../../.."))
+config_path = os.path.join(repo_root, "config.ini")
+
+if not os.path.exists(config_path):
+    raise FileNotFoundError(f"Config file not found: {config_path}")
+
+config = configparser.ConfigParser()
+config.read(config_path)
+API_TOKEN= config["EC3_API_TOKEN"]["API_TOKEN"]
```

Comment:
The new plan is to receive the API_TOKEN as a user argument that the user manually needs to input.